### PR TITLE
Model Evaluation and Diagnostics Tools list

### PR DESCRIPTION
**Pull request by rbeucher**
I have started to add links to the existing MED tools

### CODE REVIEW

File: docs/Community/MED.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+The framework is suited to analysing any MOM5/MOM6 output, as well as output from other models.
+
+The cookbook is structured as follows:
+
+- The cosima-cookbook repository includes boiler-plate code and scripts that underpin the cookbook.
+- The cosima-recipes repository includes example notebooks on which you can base your analyses.
+
+
+## [METPLUS][metplus-web]
```
Comment (aidanheerdegen) [inline docs/Community/MED.md:46]: For consistency this should be METplus

File: docs/Community/MED.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+[**Source Code**][ilamb-source]
+
+The International Land Model Benchmarking (ILAMB) project is a model-data intercomparison and integration project designed to improve the performance of land models and, in parallel, improve the design of new measurement campaigns to reduce uncertainties associated with key land surface processes.
+
+[**iLAMB-Data**][ilambdata-source]:
+A collection of scripts used to format ILAMB data and community portal to make contributions
+
+## [COSIMA Cookbook / Recipes][cosimacb-web]
+
+**Domain**: Ocean
```
Comment (aidanheerdegen) [inline docs/Community/MED.md:29]: Are there 3 (4?) domains? Atmosphere, Land, Ocean (Lithosphere?)? So sea-ice and land-ice aren't domains, but sub-components in those domains? Asking for a friend.

File: docs/Community/MED.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+- METexpress: [sources][METexpress-doc] | [docs][METexpress-source]
+- METplotpy: [sources][METplotpy-doc] | [docs][METplotpy-source]
+- METcalcpy: [sources][METcalcpy-doc] | [docs][METcalcpy-source]
+- METdatadb: [sources][METdatadb-doc] | [docs][METdatadb-source]
+
+
+## [PCMDI Metrics Package (PMP)][pmp-doc]
+
+[**Documentation**][pmp-doc] |  
+[**Sources**][pmp-source]
```
Comment (aidanheerdegen) [inline docs/Community/MED.md:72]: This has a line-break when rendered. Not sure why.

File: docs/Community/MED.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+[**Cookbook Source Code**][cosimacb-source]
+[**Cookbook Recipes**][cosimacb-recipes]
+
+The COSIMA (Consortium for Ocean-Sea Ice Modelling in Australia) Cookbook / Recipes is a framework for analysing output from ocean-sea ice models. The focus is on the ACCESS-OM2 suite of models being developed and run by members of COSIMA: Consortium for Ocean-Sea Ice Modelling in Australia.
+
+The framework is suited to analysing any MOM5/MOM6 output, as well as output from other models.
+
+The cookbook is structured as follows:
+
+- The cosima-cookbook repository includes boiler-plate code and scripts that underpin the cookbook.
```
Comment (aidanheerdegen) [inline docs/Community/MED.md:42]: "Including a database for exploring and loading available data"

**Review by aidanheerdegen — APPROVED** [review]
Looks great!

File: docs/Community/MED.md (hunk 2 above)
Comment (rbeucher) [reply docs/Community/MED.md:29]: Tricky one. I tend to think Ocean, Atmosphere, Land, Ice but Ice is really a sub-component, yes.

Comment (aidanheerdegen) [reply docs/Community/MED.md:29]: I was wondering if the domain should be Ocean/Sea-Ice, but it's not important. Really it's about discovery and helping users find the right thing, so maybe it makes sense to mention sea-ice, but could list as sub-domain. No idea if sea-ice people would be annoyed by that or not.

**Pull request merged**